docs/pages/database-access/guides/aws-cassandra-keyspaces.mdx

```diff
@@ -0,0 +1,200 @@
+  --auth-server=mytenant.teleport.sh:443 \
+  --name=keyspaces \
+  --protocol=cassandra \
+  --aws-account-id=12345678912 \
+  --aws-region=us-east-1 \
+  --labels=env=dev
+```
+
+</ScopedBlock>
+
+<Admonition type="tip">
+  You can start the Database Service using a configuration file instead of CLI flags.
+  See the [YAML reference](../reference/configuration.mdx) for details.
+</Admonition>
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+
+## Step 3/5. [Optional] Create AWS Keyspaces role:
+Go to the IAM -> Access Management -> [Roles](https://console.aws.amazon.com/iamv2/home#/roles).
+Press Create Role.
+
+![Create Role Step 1](../../../img/database-access/guides/keyspaces/create-role-step1.png)
+Create custom [IAM Keyspaces Policy](https://docs.aws.amazon.com/keyspaces/latest/devguide/security_iam_id-based-policy-examples.html) or select already existed `AmazonKeyspacesReadOnlyAccess` policy.
+![Create Role Step 1](../../../img/database-access/guides/keyspaces/create-role-step2.png)
+<Admonition type="tip">
+  You can make the policy more strict by providing specific IAM role cassandra action and specifying the cassandra keyspace and table [Amazon Keyspaces identity-based policy examples](https://docs.aws.amazon.com/keyspaces/latest/devguide/security_iam_id-based-policy-examples.html).
+</Admonition>
+Confirm role
+Enter a role name and press create role.
+![Create Role Step 1](../../../img/database-access/guides/keyspaces/create-role-step3.png)
+
+
+## Step 4/5. Give Teleport permissions to assume roles
+
+Next, attach the following policy to the IAM role or IAM user the Teleport
+Database Service instance is using, which allows the Database Service to
+assume the IAM roles:
+
+```yaml
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "sts:AssumeRole",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+<Admonition type="tip">
+  You can make the policy more strict by providing specific IAM role resource
+  ARNs in the "Resource" field instead of using a wildcard.
+</Admonition>
+
+
+## Step 5/5. Connect
+
+Once the Database Service has joined the cluster, log in to see the available
+databases:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+  ```code
+  $ tsh login --proxy=teleport.example.com --user=testuser
+  $ tsh db ls
+  # Name      Description Allowed Users Labels  Connect
+  # --------- ----------- ------------- ------- -------
+  # keyspaces             [*]           env=dev
+  ```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+  ```code
+  $ tsh login --proxy=mytenant.teleport.sh --user=testuser
+  $ tsh db ls
+  # Name      Description Allowed Users Labels  Connect
+  # --------- ----------- ------------- ------- -------
+  # keyspaces             [*]           env=dev
+  ```
+
+</ScopedBlock>
+
+To connect to a particular database instance :
+```code
+$ tsh db connect --db-user=KeyspacesReader keyspaces
```

Comment:
If not explained already in previous steps, we should note here the `KeyspacesReader` is the AWS role name.